python/ml4ir/applications/ranking/tests/data/configs/feature_config_aux_loss.yaml

```diff
@@ -0,0 +1,155 @@
+query_key:
+  name: query_id
+  node_name: query_id
+  trainable: false
+  dtype: string
+  log_at_inference: true
+  feature_layer_info:
+    type: numeric
+    shape: null
+  serving_info:
+    name: queryId
+  tfrecord_type: context
+rank:
+  name: rank
+  node_name: rank
+  trainable: false
+  dtype: int64
+  log_at_inference: true
+  feature_layer_info:
+    type: numeric
+    shape: null
+  serving_info:
+    name: originalRank
+    default_value: 0
+  tfrecord_type: sequence
+label:
+  name: clicked
+  node_name: clicked
+  trainable: false
+  dtype: int64
+  log_at_inference: true
+  feature_layer_info:
+    type: numeric
+    shape: null
+  serving_info:
+    name: clicked
+  tfrecord_type: sequence
+features:
+  - name: text_match_score
+    node_name: text_match_score
+    trainable: true
+    dtype: float
+    log_at_inference: true
+    is_aux_label: true
```

Comment:
@mohazahran FYI - this is a major change compared to earlier. There is no separate `aux_label` key anymore. Replaced with a sub field called `is_aux_label`. This simplified some of the TFRecord parsing logic IMO.